mslib/msui/mpl_qtwidget.py

```diff
@@ -926,11 +927,14 @@ def set_waypoints_model(self, model):
             # Create a path interactor object. The interactor object connects
             # itself to the change() signals of the flight track data model.
             appearance = self.get_map_appearance()
-            self.waypoints_interactor = mpl_pi.HPathInteractor(
-                self.map, self.waypoints_model,
-                linecolor=appearance["colour_ft_vertices"],
-                markerfacecolor=appearance["colour_ft_waypoints"])
-            self.waypoints_interactor.set_vertices_visible(appearance["draw_flighttrack"])
+            try:
+                self.waypoints_interactor = mpl_pi.HPathInteractor(
+                    self.map, self.waypoints_model,
+                    linecolor=appearance["colour_ft_vertices"],
+                    markerfacecolor=appearance["colour_ft_waypoints"])
+                self.waypoints_interactor.set_vertices_visible(appearance["draw_flighttrack"])
+            except IOError as err:
```

Comment:
Why IOError of all exceptions...?

Reply:
Input/Output for now.

another wish?

Reply:
I am just wondering what kind of condition would raise this kind of error within that class. I guess the Exception is there for a reason. No action necessary.

Reply:
see at the change in mslib/msui/mpl_pathinteractor.py

```
if len(wp_vertices) == 0:
   raise IOError("mscolab session expired")
```

```
if wp_vertices is None:
￼            wp_vertices = self.pathpatch.get_path().wp_vertices
￼            if len(wp_vertices) == 0:
￼                raise IOError("mscolab session expired")
```

Reply:
this is also something we maybe have to discuss/refactor. 
we have some places where we have an empty list or None .